src/github/middleware.ts

```diff
@@ -71,7 +71,7 @@ function extractWebhookEventNameFromContext(context: CustomContext<any>): string
 
 // TODO: fix typings
 export default (
-	callback: (context: CustomContext, jiraClient: JiraClient, util: any) => Promise<void>
+	callback: (context: CustomContext, jiraClient: JiraClient, util: any, githubInstallationId: number) => Promise<void>
```

Comment:
@ccrolf this is where I pass the `installationId` into the callback function that is wrapped by the middleware